src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -1,7 +1,6 @@
-
 #!/bin/bash
 
-if [ ${container_dbg} ] ;
+if [ ${LOOP_ON_FAIL} ] ;
```

Comment:
Better decide what is the value we are looking for in all places:
```suggestion
if [ "$LOOP_ON_FAIL" = "true" ] ;
```